robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -198,6 +209,35 @@ def get_next_payment_date_number(self, paynum):
         newString = f"{dd}/{mm}/{tokens[2]}"
         return newString
 
+    @capture_screenshot_on_error
+    def validate_current_and_next_year_values(self, amount):
+        """Takes in the parameter current installment payment (amount)
+        calculates the current and next year value payments and
+        validates them with the values displayed on the UI. """
+        installmentrow = npsp_lex_locators["erd"]["installment_row"]
+        installments = self.selenium.get_webelements(installmentrow)
+        count = len(installments)
+        print(f"Number of installments created is {count}")
+        i = 1
+        curr_year_value = 0
+        next_year_value = 0
+        values = {}
+        while i < count:
+            datefield = npsp_lex_locators["erd"]["installment_date"].format(i)
+            installment_date = self.selenium.get_webelement(datefield)
+            actual_date = self.selenium.get_webelement(installment_date).text
+            year = datetime.strptime(actual_date, "%m/%d/%Y").year
+            curr_year = datetime.now().year
+            next_year = (datetime.now() + relativedelta(years=1)).year
+            if curr_year == year:
+                curr_year_value = curr_year_value + int(amount)
+            if next_year == year:
+                next_year_value = next_year_value + int(amount)
```

Comment:
Should this be if else ? I don't see how these both could be true at the same time

Reply:
Changed to else